quickjs.h

```diff
@@ -516,6 +516,16 @@ typedef struct JSClassDef {
     JSClassExoticMethods *exotic;
 } JSClassDef;
 
+#define JS_EVAL_OPTIONS_VERSION 1
+
+typedef struct JSEvalOptions {
+  int version;
+  const char *filename;
+  int eval_flags;
+  int line_num;
+  // can add new fields in ABI-compatible manner by incrementing JS_EVAL2_VERSION
+} JSEvalOptions;
+
```

Comment:
`filename` should be moved to first or third position to avoid padding on 64-bit targets

Reply:
Not the first field, that should be the version.

Reply:
done